src/button/_button.scss

```diff
@@ -14,292 +14,186 @@
  * limitations under the License.
  */
 
-@import "../variables";
-@import "../mixins";
-
-// The button component. Defaults to a flat button.
 .mdl-button {
-  background: transparent;
+  min-height: 36px;
```

Comment:
Are the 36px only used here, or are they in any implicit calculations elsewhere in the component? For the former, I'm happy to have the value added straight in, but for the latter I'd prefer if we still use a variable, even if we keep it local to this file.